app/controllers/onboardings_controller.rb

```diff
@@ -0,0 +1,5 @@
+class OnboardingsController < ApplicationController
```

Comment:
should this be a singular name, `OnboardingController` ? It's declared as a `resource`, not `resources` in the routes file